PatternPal/PatternPal.Core/Checks/ClassCheck.cs

```diff
@@ -1,15 +1,21 @@
-namespace PatternPal.Core.Checks;
+﻿namespace PatternPal.Core.Checks;
 
 internal class ClassCheck : CheckBase
 {
     private readonly IEnumerable< ICheck > _checks;
 
+    //A list of all found instances
+    internal List<INode> MatchedEntities { get; private set; }
```

Comment:
Ik kan even niet zo goed achterhalen wat het doel is van deze lijst, ik zie hem eigenlijk alleen terug komen in de tests.